qaframework-bdd-tests/src/main/java/org/openmrs/contrib/qaframework/page/AdministrationManageRolesPage.java

```diff
@@ -0,0 +1,48 @@
+/**
+ * This Source Code Form is subject to the terms of the Mozilla Public License,
+ * v. 2.0. If a copy of the MPL was not distributed with this file, You can
+ * obtain one at http://mozilla.org/MPL/2.0/. OpenMRS is also distributed under
+ * the terms of the Healthcare Disclaimer located at http://openmrs.org/license.
+ * 
+ * Copyright (C) OpenMRS Inc. OpenMRS is a registered trademark and the OpenMRS
+ * graphic logo is a trademark of OpenMRS Inc.
+ */
+package org.openmrs.contrib.qaframework.page;
+
+import org.openmrs.contrib.qaframework.helper.Page;
+import org.openqa.selenium.By;
+
+public class AdministrationManageRolesPage extends Page{
+
+	protected static final String PAGE_URL = "/openmrs/admin/users/role.list";
+	private static final By ADD_NEW_ROLE = By.cssSelector("#content > a");
```

Comment:
is a `ADD_NEW_ROLE` a button, link, field or what?

Reply:
it's a link